### PR TITLE
Feature/workshop 592698898

### PR DESCRIPTION
**Pull request by helderjordao-dtx**
Test Case:

- Verify that added PR templates;
- Verify project description at README.md;

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# workshop
\ No newline at end of file
+# workshop
+
+- This project is a pilot to describe development wo[f]kflow.
```
Comment (Luis-cruz-DTx) [inline README.md]: correct typo on `workflow.`

**Review by Luis-cruz-DTx — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (helderjordao-dtx) [reply README.md]: thanks for your review, i will change it.

**Review by Luis-cruz-DTx — APPROVED** [review]
Changes were done, code seems fine.

**Review by paulojnaraujo — APPROVED** [review]
Since this is an important issue I will approve it. New issues should be created if needed.

**Pull request merged**